src/extras/MovieClip.js

```diff
@@ -56,13 +56,13 @@ function MovieClip(textures)
     this.onComplete = null;
 
     /**
-     * The MovieClips current frame index (this may not have to be a whole number)
+     * Elapsed time since animation has been started, used internally to display current texture
```

Comment:
@gprzybylowicz this is a a convention in JavaScript in general. Marking with private and putting an underscore is means it is internal, and does not merrit a major version increase if the name changes, or any special treatment in deprecation etc..